addons/udes_stock_picking_batch/tests/test_stock_picking_batch.py

```diff
@@ -1374,3 +1388,1127 @@ def _create_valid_batch(self):
         )
 
         return picking, Batch.create_batch(self.picking_type_pick.id, None)
+
+    def test_unpickable_item_single_move_line_success_default_type(self):
+        """
+        Tests that the picking is confirmed and an stock investigation transfer
+        is created if a picking type is not specified. The picking remains
+        confirmed because there isn't more stock available.
+        """
+        picking, batch = self._create_valid_batch()
+        move_line = picking.move_line_ids[0]
+        reason = 'missing item'
+        batch.unpickable_item(package_name=move_line.package_id.name,
+                              reason=reason)
+        unpickable_picking = self.package_one.find_move_lines().picking_id
+
+        self.assertEqual(picking.state, 'confirmed',
+                         'picking was not confirmed')
```

Comment:
Nit: We can probably drop the assert messages, makes it clearer. 
'stock investigation picking type not set by unpickable_item' isn't as clear as unpickable picking type does not equal internal picking type